### PR TITLE
minor note: added

### PR DESCRIPTION
**Pull request by TheBigBear**
note based on findings in #25360

### CODE REVIEW

File: salt/states/file.py [hunk 1]
```diff
@@ -2490,6 +2490,9 @@ def replace(name,
             - pattern: |
                 CentOS \(2.6.32[^\n]+\n\s+root[^\n]+\n\)+
     '''
```
Comment (jfindlay) [inline salt/states/file.py]: This line needs to be removed.

**Comment by TheBigBear** [comment]
@jfindlay sorry, I don't understand why you say "this line needs to be removed?" which line? line 2491? Have I got the 'rst' syntax wroing for a Note: box?

**Comment by jfindlay** [comment]
@TheBigBear, you added the `note` after the end of the doc string, resulting in too many `'''`.

**Comment by TheBigBear** [comment]
@jfindlay thanks for the explanation. So where do I best read-up to learn about the docstring? (python or salt) I guess this is not really github web doc 'rst' markup related , as I thought it was. 
Is there ever a way to do a 'preview' of the edits of the doc? The previiew button just shows a text side by side diff view, but I can't find a preview of the rendered html file? So I was flying 'blind' so to speak.

**Comment by jfindlay** [comment]
@TheBigBear, it's not a big deal.  I was only trying to be helpful. :-)  There are several things going on here that may not be obvious.  In python there is a special multiline string construct beginning and ending either with `'''` or `"""`.  These string literals are commonly used for and referred to as documentation strings when they are placed at the beginning of file, class, and function definitions.  The salt documentation generator automatically looks for these strings and uses them to construct the reference documentation.

**Comment by TheBigBear** [comment]
Thanks for the explanations, I find there is much to be learned here .... ;-) Appreciate you taking the time to explain a little of it.

**Comment by jfindlay** [comment]
@TheBigBear, no problem.  Let me know if you run into any other difficulties.

**Comment by jfindlay** [comment]
@TheBigBear, one more minor issue, salt docs use [rest](http://docutils.sourceforge.net/rst.html) rather than [markdown](http://daringfireball.net/projects/markdown/), which is used by github.  Code quotations in rest are delimited by ```` rather than ````` or ``` as is used by markdown.

**Pull request merged**